lib/controllers/foods.js

```diff
@@ -26,17 +31,21 @@ const create = (request, response, next) => {
   Food.createFood(name, calories)
     .then( (data) => {
       if (!data.rows[0]) { return response.sendStatus(400) }
-      response.status(201).json(data.rows)
+      response.sendStatus(201).json(data.rows)
+    }).catch( (error) => {
+      return response.sendStatus(500)
     })
 }
 
 const destroy = (request, response, next) => {
   let id = request.params.id
   Food.deleteFood(id)
     .then( (data) => {
-      eval(pry.it)
-      // if (!data.rows[0]) { return response.sendStatus(404) }
-      // response.json(data.rows)
+      if (data === 0) { return response.sendStatus(404) }
+      response.sendStatus(201)
+    }).catch( (error) => {
+      return response.sendStatus(500)
+      // return response.sendStatus(409)
     })
```

Comment:
@kat3kasper We are wondering about error catching some of these. For strange errors (like passing a string name as an id, or something like that) We could return more specific errors (see commented out lines), but are currently leaning towards providing less information in the form of a generic 500 server error. What are your thoughts on this?